maquetacion/src/scss/_form.scss

```diff
@@ -0,0 +1,28 @@
+@import 'colors';
+@import 'mixins';
+
+.form-input {
+  display: flex;
+  flex-direction: column;
+  margin: 8px 0;
+  width: 100%;
+
+  &__label {
+    font-family: $primary-font;
+    font-size: 16px;
+    line-height: 24px;
+    margin-bottom: 8px;
+  }
+
+  &__input {
+    border: none;
+    border-radius: 5px;
+    height: 32px;
+  }
+}
+
+.form-divider {
```

Comment:
This class and the `hr` element are not really needed. The way to achieve this without any new element is to add a `border-bottom` to the upper element (in this case, the login button)

Reply:
Solved with a form container